hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -112,20 +113,14 @@ public H2Dialect(DialectResolutionInfo info) {
 	}
 
 	public H2Dialect() {
-		this( SimpleDatabaseVersion.ZERO_VERSION );
+		this( MINIMUM_VERSION );
 	}
 
 	public H2Dialect(DatabaseVersion version) {
 		super(version);
 
 		// https://github.com/h2database/h2database/commit/b2cdf84e0b84eb8a482fa7dccdccc1ab95241440
-		limitHandler = version.isSameOrAfter( 1, 4, 195 )
-				? OffsetFetchLimitHandler.INSTANCE
-				: LimitOffsetLimitHandler.INSTANCE;
-
-		if ( version.isBefore( 1, 2, 139 ) ) {
-			LOG.unsupportedMultiTableBulkHqlJpaql( version.getMajor(), version.getMinor(), version.getMicro() );
-		}
+		limitHandler = OffsetFetchLimitHandler.INSTANCE;
```

Comment:
Please remove the field and simply return `OffsetFetchLimitHandler.INSTANCE` from the `getLimitHandler` method.